### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by paltusplintus**
please review a small bugfix. 
Think we can use the 
export_dbase_json/import_json_data for test cases

i.e. create data with 
from model_managers import ModelManager
from data_loaders import FileDataLoader
from model_appliers import ModelApplier
as needed
then 
export_dbase_json
save in json file and use the file in tests with import_json_data

### CODE REVIEW

**Comment by paltusplintus** [comment]
@shannonhaughton could you please review this PR? Thanks.

File: neointerface/neointerface.py [hunk 1]
```diff
@@ -1831,15 +1831,21 @@ def import_json_data(self, json_str: str):
...
 
         try:
-            json_list = json.loads(json_str)  # Turn the string (representing a JSON list) into a list
+            json_obj = json.loads(json_str)  # Turn the string (representing a JSON list) into a list
         except Exception as ex:
             raise Exception(f"Incorrectly-formatted JSON string. {ex}")
 
+        if isinstance(json_obj, list):
+            json_list = json_obj
+        elif isinstance(json_obj, dict):
```
Comment (shannonhaughton) [inline neointerface/neointerface.py:1840]: I dont think this case would ever happen with the code as it is now, as if json_obj is a dict it fails on line 1834 
image

Comment (shannonhaughton) [reply neointerface/neointerface.py:1840]: this means you currently cant use the output of export_dbase_json as an input for this

**Comment by paltusplintus** [comment]
closing as outdated

**Pull request closed without merging**